### PR TITLE
<ChevronDownIcon>, <ThumbsUpIcon>, <ThumbsDownIcon> and <Send>

### LINKED ISSUE #99: Add more icons
- [x] Sign out
- [ ] Like
- [ ] Dislike
- [ ] Submit
- [ ] Cancel
- [ ] Reply

### PR DESCRIPTION
**Pull request by axross**
Closes #99.

### CODE REVIEW

**Comment by axross** [comment]
@ryotogashi Can you have a look at these updates when you have time?

File: components/Icon/Icon.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        onClick={() => setActive(!isActive)}
+        style={{
+          display: "flex",
+          alignItems: "center",
+          color: isActive ? "#ff6b6b" : "#0f151c",
+          transition: "color 200ms ease",
+          cursor: "pointer"
+        }}
+      >
+        {"Click to change "}
```
Comment (ryotogashi) [inline components/Icon/Icon.stories.tsx]: @axross Do you need this?
`{"Click to change "}`
Or should it be below?
`Click to change`

File: components/Icon/Icon.tsx [hunk 2]
```diff
@@ -7,21 +8,54 @@ interface Props extends React.Attributes {
...
+      <path d="M10 15v4a3 3 0 0 0 3 3l4-9V2H5.72a2 2 0 0 0-2 1.7l-1.38 9a2 2 0 0 0 2 2.3zm7-13h2.67A2.31 2.31 0 0 1 22 4v7a2.31 2.31 0 0 1-2.33 2H17"></path>
+    </Root>
+  );
+}
+
+export function SendIcon(props: Props) {
+  return (
+    <Root xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" {...props}>
+      <line x1="22" y1="2" x2="11" y2="13"></line>
+      <polygon points="22 2 15 22 11 13 2 9 22 2"></polygon>
```
Comment (ryotogashi) [inline components/Icon/Icon.tsx]: @axross you can change
```suggestion
      <line x1="22" y1="2" x2="11" y2="13" />
      <polygon points="22 2 15 22 11 13 2 9 22 2" />
```

File: components/Icon/Icon.tsx [hunk 3]
```diff
@@ -7,21 +8,54 @@ interface Props extends React.Attributes {
...
+    </Root>
+  );
+}
+
+export function ChevronDownIcon(props: Props) {
+  return (
+    <Root xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" {...props}>
+      <polyline points="6 9 12 15 18 9"></polyline>
```
Comment (ryotogashi) [inline components/Icon/Icon.tsx]: @axross same here
```suggestion
      <polyline points="6 9 12 15 18 9" />
```

File: components/Icon/Icon.tsx (hunk 2 above)
Comment (axross) [reply components/Icon/Icon.tsx]: That's right. I missed it out. Thanks!

File: components/Icon/Icon.tsx (hunk 3 above)
Comment (axross) [reply components/Icon/Icon.tsx]: Thanks here too!

File: components/Icon/Icon.stories.tsx (hunk 1 above)
Comment (axross) [reply components/Icon/Icon.stories.tsx]: That's intentional. Because the parent element is grid and not let to split text nodes

**Comment by axross** [comment]
Thanks!

**Pull request merged**